src/twisted/test/test_defer.py

```diff
@@ -680,6 +680,18 @@ def test_maybeDeferredSyncError(self):
         self.assertEqual(len(E), 1)
         self.assertEqual(str(E[0].value), expected)
 
+    def test_maybeDeferredSyncFailure(self):
+        """
+        L{defer.maybeDeferred} should catch exception raised by a synchronous
+        function and errback its resulting L{defer.Deferred} with it.
```

Comment:
It looks like this docstring is the same as the one for `test_maybeDeferredSyncException` and doesn't match the body of the new test.

Reply:
whoops

Reply:
fixed.